.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+
+env:
+  FORCE_COLOR: 2
+  NODE: 14
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "${{ env.NODE }}"
+          cache: npm
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Test
+        run: npm test
+
+      - name: Upload docs
+        uses: actions/upload-artifact@v2
+        if: github.repository == 'jakearchibald/svgomg' && github.ref == 'refs/heads/main'
+        with:
+          name: docs
+          path: ./build/
+          if-no-files-found: error
```

Comment:
What's this doing?

Reply:
I basically keep the build files from the test step, upload them as a zip and then download and extract them in the deploy step (instead of rebuilding).

Reply:
You can see this in action here for example: https://github.com/twbs/blog/runs/3535610220

Reply:
Aha, the renaming makes it clearer!